app/components/primer/alpha/nav_list.ts

```diff
@@ -120,6 +158,70 @@ class NavListElement extends HTMLElement {
     template.innerHTML = html
     return document.importNode(template.content, true)
   }
+
+  private findSelectedNavItemById(itemId: string): HTMLElement | null {
```

Comment:
Nit: `private` is only a TS convention. If you prefix with `#` it's truly private. So `#findSelectedNavItemById`.